docs/cep/query-guide/functions/json/isExists.md

```diff
@@ -20,21 +20,54 @@ Function checks whether there is a JSON element present in the given path or not
 ## Example 1
 
 ```sql
-@info(name = 'query1')
 json:isExists(json, '$.name')
 ```
 
-This query, named 'query1', uses the `json:isExists` function to check if an element exists in the given JSON path. In this case, the function checks for the existence of the `$.name` element.
-
-When the input `json` has the format `{'name' : 'John', 'age' : 23}`, the function returns `true`, as there is an element in the given path.
+Given a JSON object like `{'name' : 'John', 'age' : 23}`, the function `json:isExists(json,'$.name')` checks if an element exists at the key `name`. In this instance, it returns `true` because there is a matching element at `$.name`.
 
 ## Example 2
 
 ```sql
-@info(name = 'query1')
 json:isExists(json, '$.salary')
 ```
 
-This query, named 'query1', uses the `json:isExists` function to check if an element exists in the given JSON path. In this case, the function checks for the existence of the `$.salary` element.
+With a JSON object like `{'name' : 'John', 'age' : 23}`, the function `json:isExists(json,'$.salary')` checks if an element exists at the key `salary`. In this instance, it returns `false` because there is no matching element at `$.salary`.
+
+## Example 3
+
+```sql
+json:isExists(json, '$.age')
+```
+
+Given a JSON object like `{'name' : 'John', 'age' : 23}`, the function `json:isExists(json,'$.age')` checks if an element exists at the key `age`. In this instance, it returns `true` because there is a matching element at `$.age`.
+
+## Example 4
+
+```sql
+json:isExists(json, '$.address.city')
+```
+
+For a JSON object like `{'name' : 'John', 'address' : {'city' : 'NY', 'country' : 'USA'}}`, the function `json:isExists(json,'$.address.city')` checks if an element exists at the key `address.city`. In this instance, it returns `true` because there is a matching element at `$.address.city`.
+
+## Example 5
+
+```sql
```

Comment:
@koshy-thomas and @Luba89, also an issue with this one.

Reply:
@diana-macrometa this is also the same you need to change boolean to bool in:
CREATE SINK STREAM AddressExistsStream (name string, addressExists bool);